napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,103 @@
+import warnings
+from typing import Union, List
+
+import numpy as np
+from napari.layers import Image, Labels
+from vispy.color import Colormap
+
+
+def stack_to_images(
+    stack: Union[Image, Labels],
+    axis: int,
+    colormaps: List[Union[str, Colormap]] = None,
+) -> List[Image]:
+    """Function to split the active layer into separate layers along an axis
+
+    Parameters
+    ----------
+    stack : napari.layers.Image
+        The image stack to be split into a list of image layers
+    axis : int
+        The axis to split along.
+
+    Returns
+    -------
+    list
+        List of images
+    """
+
+    data = stack.data
+    name = stack.name
+    num_dim = len(data.shape)
+
+    if num_dim < 3:
+        warnings.warn(
+            "The image needs more than 2 dimensions for splitting", UserWarning
+        )
+        return None
+
+    if axis >= num_dim:
+        warnings.warn(
+            "Can't split along axis {}. The image has {} dimensions".format(
+                axis, num_dim
+            ),
+            UserWarning,
+        )
+        return None
+
+    cmap = stack.colormap
+    if colormaps is None:
+        colormaps = data.shape[axis] * [cmap]
+
+    imagelist = list()
+
+    for i in range(data.shape[axis]):
+        layer_name = "{:02d}_{}".format(i, name)
```

Comment:
@cwood1967 although I understand the desire to keep the channel information out front, it really doesn't gel with convention. Ultimately, I think this is a usability fix that needs to happen on how we display layer names, but in my opinion the image number should appear at the end. I'd propose here:

```suggestion
        layer_name = f'{name} layer {i}'
```

or

```python
        layer_name = f'{name} ch{i}'
```

for example.